akka-actor/src/main/mima-filters/2.5.x.backwards.excludes

```diff
@@ -0,0 +1,6 @@
+ProblemFilters.exclude[MissingClassProblem]("akka.actor.Inbox$")
```

Comment:
Wow does this work? `2.5.x.backwards.excludes`?

Reply:
Yes but I think it should be 2.6.x instead of 2.5.x

Reply:
Or not!